collector/container/Dockerfile.ubi

```
@@ -31,12 +27,18 @@ RUN dnf -y update && \
         libtool \
         git \
         elfutils-libelf-devel \
-        gtest-devel \
-        gmock-devel \
-        jsoncpp-devel  \
         tbb-devel \
-        jq-devel \
         c-ares-devel && \
+    dnf config-manager --enable codeready-builder-for-rhel-8-x86_64-rpms && \
+    dnf -y install \
+        gmock-devel \
+        gtest-devel && \
+    dnf config-manager --enable amq-clients-2-for-rhel-8-x86_64-rpms && \
+    dnf -y install \
+        jsoncpp-devel && \
+    dnf config-manager --enable rhocp-4.6-for-rhel-8-x86_64-rpms && \
+    dnf -y install \
+        jq-devel && \
```

I like this, it makes it quite clear which content sets are required for which packages. Would it make sense to use the `--enablerepo` flag in the install commands so the scope is even more clear?
```suggestion
    dnf -y --enablerepo codeready-builder-for-rhel-8-x86_64-rpms  install \
        gmock-devel \
        gtest-devel && \
    dnf -y --enablerepo amq-clients-2-for-rhel-8-x86_64-rpms install \
        jsoncpp-devel && \
    dnf -y --enablerepo rhocp-4.6-for-rhel-8-x86_64-rpms  install \
        jq-devel && \
```

---

Done, thanks!